### ISSUE TITLE
Uncontrolled parts lock up when wheels are attached

### ISSUE THREAD

**Issue by Nimnu** [MagicSmokeIndustries/InfernalRobotics#112]
This is a mouthful to say so here goes.
1. Attaching any wheel to an uncontrolled IR part, directly or via other parts, locks that part and orientation to whatever was configured in the SPH/VAB.
2. Nested uncontrolled parts, connected directly or via other parts, continue to work but do not affect the 'locked' parts orientation.  
3. Uncontrolled parts that rotate in a different plane/orientation do not seem to be affected.

Example CRAFT file (hopefully) attached.
[IR Test 01.zip](https://github.com/MagicSmokeIndustries/InfernalRobotics/files/265325/IR.Test.01.zip)

Edit:  Using stock KSP 1.1.2.1260 x64 & IR 2.0.4

**Comment by Nimnu** [comment]
Just found something new, if you give it a 'kick' by pressing the < or > buttons in the IR Editor Window it will start to behave normally again.  I suspect something in the KSP wheel code is conflicting with how uncontrolled IR parts behave when physics starts.

**Comment by ZiwKerman** [comment]
Yes, it is the wheel auto-strutting getting in the way. In our latest versions we only disable these auto-struts when the servos are actively moved and re-enable them when the movement is stopped. When you command the uncontrolled servo to move (which is not a normal practice), this command basically never finishes. Apart from completely disabling auto-strutting there is little we can do to work around it for uncontrolled parts.

**Comment by Nimnu** [comment]
With the auto-strut changes in 1.2 this seems to be resolved.
